client/apps/settings/client/curations/display/components/canvas.jsx

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+
+import CanvasImages from './canvas_images.jsx'
+import CharacterLimitInput from 'client/components/character_limit/index.jsx'
+
+export default class Canvas extends React.Component {
+  render () {
```

Comment:
If there's only a `render` method I prefer the use of stateless functional components